pkg/server/backuprestoreserver.go

```diff
@@ -275,17 +275,26 @@ func (b *BackupRestoreServer) runEtcdProbeLoopWithSnapshotter(ctx context.Contex
 			}
 		}
 
-		// set server's healthz endpoint status to OK so that
-		// etcd is marked as ready to serve traffic
-		handler.SetStatus(http.StatusOK)
-
 		if runServerWithSnapshotter {
 			if b.ownerChecker != nil {
 				// Check if the actual owner ID matches the expected one
 				// If the check fails or returns false, take a final full snapshot if needed
```

Comment:
Adapt this comment to the new changes appropriately.

Reply:
I didn’t get it ?

Reply:
just wanted to set the readinessProbe only after `ownercheck` succeed/fail . Previously I missed to test this [PR](https://github.com/gardener/etcd-backup-restore/pull/411) with ownercheck.

Reply:
I guess I wasn't very clear since my comment is on a line that wasn't changed in the PR. I meant that 
```suggestion
				// If the check returns false, take a final full snapshot if needed. If the check returns an error continue with normal operation.
```